addon/content/quoting.js

```diff
@@ -201,6 +201,39 @@ class _Quoting {
       }
     }
   }
+
+  /**
+   * Use heuristics to find common types of email quotes and
+   * wrap them in `<blockquote></blockquote>` tags.
+   *
+   * @param {HTMLDocument | string} doc
+   * @returns {HTMLDocument | string}
+   * @memberof _Quoting
+   */
+  normalizeBlockquotes(doc) {
+    // We want to return the same type of object that was passed to us. We allow
+    // both a string and an HTMLDom object.
+    const origType = typeof doc;
+    if (origType === "string") {
```

Comment:
Is this set-up for later, or has it been left in?

It seems strange to have both the test and code pass in a document, but have the option for handling a string.

Reply:
It's set up for later. I can remove it if you like.

I had a test for string input, but it failed because node does not have DOMParser in imported modules, for some reason...

Reply:
No problem, it is fine as-is, I was more curious.

Could you do a rebase/merge? github is showing conflicts.

Reply:
Really? It says `This branch has no conflicts with the base branch` for me...

Reply:
Ah, my fault, had it on the wrong option.